### PR TITLE
Surgery safety check 2: electric boogaloo

### PR DESCRIPTION
**Pull request by nerocavalier**
* The bug with rollerbeds should be fixed now. Additionally, I changed it to reagent_containers so you can feed, drink, and give inhalers to people now.
* [Example](https://imgur.com/a/mCKoElc)

### CODE REVIEW

**Comment by nerocavalier** [comment]
!review

**Comment by nerocavalier** [comment]
!wip

**Comment by nerocavalier** [comment]
!review

File: code/modules/surgery/surgery.dm [hunk 1]
```diff
@@ -88,8 +88,17 @@ proc/spread_germs_to_organ(var/obj/item/organ/external/E, var/mob/living/carbon/
...
 		return FALSE
+
+	var/static/list/safety_check_exceptions = list(
+		/obj/item/auto_cpr,
+		/obj/item/device/healthanalyzer,
+		/obj/item/reagent_containers,
+		/obj/item/modular_computer,
+		/obj/item/device/advanced_healthanalyzer,
+		/obj/item/device/robotanalyzer,
+		/obj/item/stack/medical)
```
Comment (Geevies) [inline code/modules/surgery/surgery.dm:101]:
```suggestion
		/obj/item/stack/medical
		)
```

**Review by Geevies — APPROVED** [review]

**Comment by nerocavalier** [comment]
Reverting doesn't seem to fix travis breaking. RIP.

**Pull request closed without merging**